proto/tendermint/rpc/grpc/types.proto

```diff
@@ -23,10 +54,56 @@ message ResponseBroadcastTx {
   tendermint.abci.ResponseDeliverTx deliver_tx = 2;
 }
 
+message BlockByHashResponse {
+  tendermint.types.Part block_part = 1;
+  bool                  is_last    = 2;
+}
+
+message BlockByHeightResponse {
+  tendermint.types.Part block_part = 1;
+  bool                  is_last    = 2;
+}
+
+message BlockMetaByHashResponse {
+  tendermint.types.BlockMeta block_meta = 1;
+}
+
+message BlockMetaByHeightResponse {
+  tendermint.types.BlockMeta block_meta = 1;
+}
+
+message CommitResponse {
+  int64                    height                = 1;
+  int32                    round                 = 2;
+  tendermint.types.BlockID block_id              = 3 [(gogoproto.nullable) = false, (gogoproto.customname) = "BlockID"];
+  repeated tendermint.types.CommitSig signatures = 4 [(gogoproto.nullable) = false];
```

Comment:
Can't we just reuse the commit proto type we already have

Reply:
45935627096293d920aa0213a85329060f950361